CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 See README.md on how to build the hipCUB documentation using Doxygen.
 
+## (Unreleased) hipCUB-2.13.1 for ROCm 6.0
```

Comment:
Please change this to ROCm 6.1

Reply:
Added changes from [https://github.com/ROCmSoftwarePlatform/rocPRIM/commit/8c9c64c2879db10b4fb5e06327cdde0f6f36b08c](https://github.com/ROCmSoftwarePlatform/rocPRIM/commit/8c9c64c2879db10b4fb5e06327cdde0f6f36b08c) and updated ROCm version to 6.1.0